SR2021/2021-01-12-team-numbers.md

```diff
@@ -0,0 +1,16 @@
+---
+to: SR2021 institutions with inactive teams
+subject: Student Robotics Teams
+---
+
+Hi,
+
+If you're reading this email, it means you have at least 1 team from your institution who seem inactive on Discord, or who have few members.
+
+If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all trying to help.
```

Comment:
Perhaps "available to help"? "trying" almost implies they're failing.
```suggestion
If you have teams requiring further guidance and support, the Discord is the best place to get support. Filled with knowledgable volunteers, competitors and team leaders, all available to help.
```

Maaybe also mention that they can email us if they're stuck but can't use Discord?